### PR TITLE
Added to wmn-data.json with SimplePlanes

### PR DESCRIPTION
**Pull request by SlopeSlayer910**
Added entry for SimplePlanes.com
{
        "name" : "SimplePlanes",
        "uri_check" : "https://www.simpleplanes.com/u/{account}",
        "e_code" : 200,
        "m_code" : 302,
        "known" : ["realSavageMan", "Jundroo", "john"],
        "cat" : "gaming"
       },
       Entry is missing its e_string and m_string as I do not yet know how to discern those

### CODE REVIEW

File: wmn-data.json [hunk 1]
```diff
@@ -4868,6 +4868,14 @@
         "known" : ["john", "bob"],
         "cat" : "images"
        },
+       {
+        "name" : "SimplePlanes",
+        "uri_check" : "https://www.simpleplanes.com/u/{account}",
+        "e_code" : 200,
+        "m_code" : 302,
```
Comment (WebBreacher) [inline wmn-data.json:4875]: Let's add an `e_string` of `<h5>joined`

Comment (WebBreacher) [inline wmn-data.json:4875]: And an `m_string` of `<title>SimplePlanes Airplanes</title>`

**Comment by WebBreacher** [comment]
Hey @SlopeSlayer910. Thanks for the submission. Finding the `e_string` and `m_string` are done by looking in the source code of the response from the server. The https://github.com/WebBreacher/WhatsMyName/blob/main/CONTRIBUTING.md file has a little bit about this.

**Comment by SlopeSlayer910** [comment]
@WebBreacher, Added the suggested `e` and `m` strings to the PR, should pass now

File: wmn-data.json (hunk 1 above)
Comment (SlopeSlayer910) [reply wmn-data.json:4875]: Sure

Comment (SlopeSlayer910) [reply wmn-data.json:4875]: Sure

Comment (WebBreacher) [reply wmn-data.json:4875]: Sorry. This is not "resolved" until you commit the e_string/m_string to this PR.

Comment (WebBreacher) [reply wmn-data.json:4875]: Sorry. This is not "resolved" until you commit the e_string/m_string to this PR.

**Comment by SlopeSlayer910** [comment]
@WebBreacher, fixed issues with local files not pushing and committing changes. Should all validate now

**Pull request merged**